### PR TITLE
Claim status add files form ceh

### PR DESCRIPTION
**Pull request by cehsu**
- make markup semantic
   - [x] document-title
   - [x] claims-upload-input-title
- delete unnused styles
    - [x] file-review
    - [x] mail-or-fax-files
    - [x] document-size-text (nested p never used)
    - [x] upload-files
 - unnest nested styles
    - [X] document-item-container (.document-title-text-container (document-size-text (p)), remove-document-button (button), &:focus) 
    - [x] claims-upload-input (h5 and usa-input-error-message)
    - [x] file-requirements (file-requirement-header, file-requirement-text)
    - [x] file-requirement-header (noted above)
    - [x] file-requirement-text (noted above)
    - [x] document-title-text-container (noted above, and need to unnest (document-size-text (p)))
    - [x] removed document-title-header
    - [x] title, renamed to document-title
    - [x] remove-document-button (noted above and unnest button or >)
  - (don't see opportunities for consolidating classes)

### CODE REVIEW

File: src/js/claims-status/components/AddFilesForm.jsx [hunk 1]
```diff
@@ -109,10 +109,10 @@ class AddFilesForm extends React.Component {
             <div className="document-title-size">
               <div className="document-title-row">
                 <div className="document-title-text-container">
-                  <div className="document-title-header">
-                    <h4 className="title">{file.name}</h4>
+                  <div>
+                    <h4 className="document-title">{file.name}</h4>
```
Comment (jbalboni) [inline src/js/claims-status/components/AddFilesForm.jsx]: I don't think this needs to be an `h4`.

File: src/js/claims-status/components/AddFilesForm.jsx [hunk 2]
```diff
@@ -90,12 +90,12 @@ class AddFilesForm extends React.Component {
         <div>
           <ErrorableFileInput
             errorMessage={this.getErrorMessage()}
-            label={<h5>Select files to upload</h5>}
+            label={<h5 className="claims-upload-input-title">Select files to upload</h5>}
```
Comment (jbalboni) [inline src/js/claims-status/components/AddFilesForm.jsx]: I don't think this should be an `h5`, that doesn't really make sense in a label

File: src/sass/claims-status.scss [hunk 3]
```diff
@@ -288,51 +288,48 @@ p.first-of-type {
...
     outline: none;
   }
 }
 
+.document-title-text-container {
+  padding-right: 1em;
+  word-break: break-all;
+}
+
+.remove-document-button {
```
Comment (jbalboni) [inline src/sass/claims-status.scss]: Should we move this class to the button element? I don't see a reason for it to be on the containing div.

**Review by jbalboni — APPROVED** [review]
LGTM, I had a couple comments about further improving some of the existing code, but your changes look good.

File: src/js/claims-status/components/AddFilesForm.jsx (hunk 2 above)
Comment (cehsu) [reply src/js/claims-status/components/AddFilesForm.jsx]: Good point @jbalboni, additional style rules have been provided to `claims-upload-input-title` to mimic the `<h5>` appearance.

File: src/js/claims-status/components/AddFilesForm.jsx (hunk 1 above)
Comment (cehsu) [reply src/js/claims-status/components/AddFilesForm.jsx]: Good point @jbalboni, additional style rules have been provided to `document-title` to mimic the `<h4>` appearance.

File: src/sass/claims-status.scss (hunk 3 above)
Comment (cehsu) [reply src/sass/claims-status.scss]: Good point @jbalboni, change made.

**Pull request merged**